schemas/address.schema.js

```diff
@@ -11,6 +11,7 @@ const addressSchema = new Schema({
     city: {type: String, required: [true, "City of address is required"]},
     parish: {type: Schema.Types.ObjectId, ref: "Parish", required: [true, "Parish of address is required"]},
     status: {type: String, enum: {values: statusEnum, message:`{VALUES} is not a valid status. statuses: ${statusEnum}`}},
+    deletedAt: {type: Date, default: Date.now}
```

Comment:
Also, deletedAt should have a default value of null. This means that it is not deleted, or an attempt to delete it has not been made. Only when the controller does a soft delete will this value have a date.